### DISCUSSION TITLE
Start Menu centering running on Windows 10

### DISCUSSION THREAD

**Discussion by Wolf-Shadow777** [valinet/ExplorerPatcher#2203] [General]
Hi, this is not a problem so much as a query. I am running windows 10, and have the taskbar centered. When pc is first started or if explorer.exe is restarted, the start menu starts on the left side and then quickly snaps centered. After which, it starts at centered after that.
Is it something that could be looked into? As I said, not so much a problem but would be nice if it worked like windows 11.

**Comment by pyrates999** [comment]
I wasn't aware that windows 10 allowed for that.  Can you show a screen shot of when the start menu is on the left side and when it is in the middle.

Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Comment by Wolf-Shadow777** [comment]
Thanks for looking at my post. And yes, win 10 does allow for this. I like the centered look and operation of win 11, but not win 11 (lol).
EP is a very small footprint compared to other software that does the same thing. Also, ep is native 64 bit.

![start left](https://github.com/valinet/ExplorerPatcher/assets/139655451/9d6d3556-0bc9-473f-95c1-a7ab72a7e1c1)
![Centered](https://github.com/valinet/ExplorerPatcher/assets/139655451/0c70a69a-7703-4f57-b967-b91bee63c4b6)
![startMenu  changes](https://github.com/valinet/ExplorerPatcher/assets/139655451/e1469115-005d-44b3-b8a0-5aad1c3098b8)
![taskbar settings](https://github.com/valinet/ExplorerPatcher/assets/139655451/c60413bb-b577-4fb0-a4fa-1581c7300750)

**Reply by pyrates999** [reply]
Your first screen shot doesn't show a taskbar.  So I can't tell if the start button has remained in the middle or not.  Can you take another screen shot?

**Reply by Wolf-Shadow777** [reply]
It does what it is supposed to do. The menu will always start on the left at startup, if exlorer.exe is refreshed either by manually doing so, or if the lid is closed and later opened (sleep).

![Screenshot_1](https://github.com/valinet/ExplorerPatcher/assets/139655451/1827be9a-faba-4ac4-9bbb-fb6a0538d321)

**Comment by pyrates999** [comment]
Thank you.  So to recreate this:

1. in the properties application for EP, set the start button and start menu to be in the middle
2. restart explorer
3. the start button will show in the middle
4. after you first press on the start button, the start menu will show briefly on the left side, then appear in the middle
5. if you press on the the start button again, the start menu will show in the middle
6. if you restart your computer
7. the start button will show in the middle
8. after you first press on the start button, the start menu will show briefly on the left side, then appear in the middle
9. if you press on the the start button again, the start menu will show in the middle
10. if your computer goes to sleep and you wake it up
11. the start button will show in the middle
12. after you press on the start button, the start menu will show on the left side and never in the middle

Is this correct?

**Reply by Wolf-Shadow777** [reply]
That is what it is doing, yes.

**Reply by pyrates999** [reply]
Good.  Now please wait for a developer to verify it.

**Comment by Amrsatrio** [comment]
This happens as well on Windows11 after I've restored the original animations. As the old menu centering is a mod not a broken Windows code, I think fixing this issue is not really worth the effort... Sorry

**Comment by mathius89** [comment]
Still hapenning to me this problem on 2025, there is any solution? Thanks!
